package/duo-pinmux/Config.in

```diff
@@ -0,0 +1,4 @@
+config BR2_PACKAGE_DUO_PINMUX
+    bool "Duo pinmux"
```

Comment:
So it could be `BR2_PACKAGE_MILKV_DUO_PINMUX` as package name and `milkv-duo-pinmux` as menu entry. As I understand, we're not going to change also tool executable name ?

Reply:
Yes, exactly. The inconsistency of the package name and executable file name is normal. And what about add more description for the package?

Reply:
Sure, I'll make it more informative about it's purpose.